vue3-apollo-quasar/src/components/RepoSubHeader/RepoSubHeader.vue

```diff
@@ -29,8 +29,8 @@
           class="row col col-md-5 col-lg-4 items-center no-wrap justify-between relative-position"
         >
           <button
-            class="git-btn menu__btn text-capitalize row justify-between items-center"
-            @click="toggleWatchMenu"
+            class="git-btn menu_btn--watch text-capitalize row justify-between items-center"
+            @click="toggleWatchMenu()"
```

Comment:
Context required, what's the convention being used here? BEM? 
Also, why do we have different button type classes: `git-btn` and `menu_btn` with different coding styles?